apps/api/src/app/admin/admin.service.ts

```diff
@@ -429,8 +429,14 @@ export class AdminService {
     };
   }
 
-  public async getUsers(): Promise<AdminUsers> {
-    return { users: await this.getUsersWithAnalytics() };
+  public async getUsers({
+    skip,
+    take = Number.MAX_SAFE_INTEGER
+  }: {
+    skip?: number;
+    take?: number;
+  }): Promise<AdminUsers> {
+    return { users: await this.getUsersWithAnalytics({ take, skip }) };
```

Comment:
Can you swap `skip` and `take` to preserve the alphabetical order please?

Reply:
Done.